docs/source/data.rst

```diff
@@ -330,3 +330,9 @@ Box mode converter
 .. autofunction:: monai.data.box_utils.get_boxmode
 .. autofunction:: monai.data.box_utils.convert_box_mode
 .. autofunction:: monai.data.box_utils.convert_box_to_standard_mode
+
+Box center
+~~~~~~~~~~
+.. autofunction:: monai.data.box_utils.box_center
```

Comment:
function name in the doc should also be updated.

Thanks.

Reply:
Thank you! I have make html and checked it works now.